webrtc.html

```diff
@@ -8621,17 +8646,26 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 <li>If <var>tones</var> contains any <a>unrecognized</a>
                 characters, <a>throw</a> an <code>InvalidCharacterError</code>.
                 </li>
-                <li>Set the object's <code><a data-for=
-                "RTCDTMFSender">toneBuffer</a></code> attribute to
-                <var>tones</var>.</li>
+                <li>Set the <code><a data-for="RTCDTMFSender">toneBuffer</a></code>
+                attribute of <var>dtmf</var> to <var>tones</var>.</li>
+                <li>Set the <a>[[\Duration]]</a> slot of <var>dtmf</var>
+                to the value of the <code><a data-for=
+                "RTCDTMFSender">duration</a></code> parameter.</li>
+                <li>Set the <a>[[\InterToneGap]]</a> slot of <var>dtmf</var>
+                to the value of the <code><a data-for=
+                "RTCDTMFSender">interToneGap</a></code> parameter.</li>
                 <li>If the value of the <code><a data-for=
                 "RTCDTMFSender">duration</a></code> parameter is less than 40,
-                set it to 40. If, on the other hand, the value is greater than
-                6000, set it to 6000.</li>
+                set the <a>[[\Duration]]</a> slot of <var>dtmf</var> to 40.</li>
+                <li>If the value of the <code><a data-for=
+                "RTCDTMFSender">duration</a></code> parameter is greater than 6000,
+                set the <a>[[\Duration]]</a> slot of <var>dtmf</var> to 6000.</li>
+                <li>If the value of the <code><a data-for=
+                "RTCDTMFSender">interToneGap</a></code> parameter is less than 30,
+                set the <a>[[\InterToneGap]]</a> slot of <var>dtmf</var> to 30.</li>
                 <li>If the value of the <code><a data-for=
-                "RTCDTMFSender">interToneGap</a></code> parameter is less than
-                30, set it to 30. If, on the other hand, the value is greater
-                than 6000, set it to 6000.</li>
+                "RTCDTMFSender">interToneGap</a></code> parameter is greater than 6000,
+                set the <a>[[\InterToneGap]]</a> slot of <var>dtmf</var> to 6000.</li>
                 <li>If <code><a data-for="RTCDTMFSender">toneBuffer</a></code>
                 is an empty string, abort these steps.</li>
                 <li>If a <em>Playout task</em> is scheduled to be run, abort
```

Comment:
[[Duration]] and [[InterToneGap]] aren't read anywhere. We should update the playout task to access them.

That said, I don't know if we need internal slots at all for [[Duration]] and [[InterToneGap]] in these times of c++ lambdas and JSImplemented code, since we're only accessing these from inside the `insertDTMF` algorithm. Perhaps, "Let *netDuration* be the *duration* parameter, no less than `30`, no more than `6000`" would suffice?